src/app/lib/utilities/srcSet/index.js

```diff
@@ -46,15 +46,14 @@ export const createSrcsets = ({
   }
 
   // eslint-disable-next-line prefer-const
-  let [primarySrcset, fallbackSrcset] = [true, false].map(isWebP =>
+  let [primarySrcset, fallbackSrcset] = [true, false].map(
```

Comment:
May just need to retain the arrow function to fix some of the broken tests: `() =>`

And below as well